docs/ossm/ossm2-migration/network-policies.md

```diff
@@ -0,0 +1,144 @@
+# OpenShift Service Mesh 2.6 migration to 3.0
```

Comment:
Maybe having the title more specific would be better here. This doc is just about network policy migration, not about full OSSM migration